recipes/magnum/all/conanfile.py

```diff
@@ -362,7 +362,7 @@ def _patch_sources(self):
                               "")
         # Get rid of cmake_dependent_option, it can activate features when we try to disable them,
         #   let the Conan user decide what to use and what not.
-        with open(os.path.join(self._source_subfolder, "CMakeLists.txt"), 'r+') as f:
+        with open(os.path.join(self._source_subfolder, "CMakeLists.txt"), 'r+', encoding="utf-8") as f:
```

Comment:
For some reason, this was failing in modern docker images (gcc11) cc/ @uilianries